server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -407,7 +407,7 @@ public void deleteSnapshot(SnapshotId snapshotId, long repositoryStateId, Action
                 Arrays.asList(snapshotFormat.blobName(snapshotId.getUUID()), globalMetaDataFormat.blobName(snapshotId.getUUID()));
             try {
                 blobContainer().deleteBlobsIgnoringIfNotExists(snapMetaFilesToDelete);
-            } catch (IOException e) {
+            } catch (Exception e) {
```

Comment:
Didn't you just make this change unnecessary by handling all exceptions and re-throwing them as `IOException`?

Reply:
Arghl, this is a leftover, I'll revert this